schedule/yam/agama_sle.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name: agama
+description: >
+  Perform interactive installation with agama.
+schedule:
+  - yam/agama/boot_agama
+  - yam/agama/patch_agama_tests
+  - yam/agama/agama
+  - yam/agama/agama_grub2
```

Comment:
can you please drop this for the rest of architectures and see if really is needed for power in sle?

Reply:
Do you mean trigger this schedule in ppc64le?

Reply:
we can use `grub_test` instead of this module, which is only useful for ppc64le, but I don't know if we hit the same problem in sle to use it. If you could create different schedule for ppc64le would be great. This belongs to https://progress.opensuse.org/issues/167024 which is unfinished, but at least we will not run it in all the architectures, typing unnecessary stuff could lead us to false positive.

Reply:
why not use agama.yaml ? https://open.qa/docs/#_exclude_modules should work to exclude those test after installation that cannot run at the moment.

Reply:
516cd3b70